pom.xml

```diff
@@ -169,6 +181,13 @@
                 <version>${version.curator}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>io.confluent</groupId>
+                <artifactId>kafka-connect-avro-converter</artifactId>
+                <!-- check new release version at https://github.com/confluentinc/schema-registry/releases -->
+                <version>2.1.0-alpha1</version>
```

Comment:
Can you also put this in a property, like maybe {{version.confluent.platform}}? If we add other dependences, we'd want to ensure the artifacts come from the same Confluent Platform version.